pkg/v1/tkg/client/validate_test.go

```diff
@@ -1092,48 +1095,236 @@ var _ = Describe("Validate", func() {
 			})
 		})
 
-		Context("Network Separation configuration and validation", func() {
-			It("should allow empty network separation configurations", func() {
-				validationError := tkgClient.ConfigureAndValidateManagementClusterConfiguration(initRegionOptions, true)
-				Expect(validationError).NotTo(HaveOccurred())
+		Context("ConfigureAndValidateAviConfiguration", func() {
```

Comment:
The changes here tests all the utility functions such as `ValidateAviControllerAccount`, `ValidateAviCloud`, etc., Can we add tests for the main validation function `ConfigureAndValidateAviConfiguration`? For example, the set of test cases added does not cover what happens when `AVI_ENABLE` is false.

Reply:
Test cases added